docs/content/en/docs/migrate/strategy/_index.md

```diff
@@ -12,5 +12,259 @@ but the format and content is still evolving.
+of the pod-scheduling phase.
+
+The `shipyard` file is a general purpose workflow engine
+that is backed by cloud events.
+It is not opinionated to a tool, platform, technology,
+or a particular "slice" of the lifecycle.
+The `TriggeredOn` property allows
+a Keptn v1 sequence to be triggered at any time
+by a user or another system.
+This capability can be used, for example,
+to trigger a data encryption and backup operation,
+or a file movement over a network, or other arbitrary activities
+that may or may not have anything to do with an application's lifecycle.
+
+When migrating to KLT,
+tasks that are not part of the lifecycle workflow
+should not be handled by KLT
+but should instead be handled by the pipeline engine tools being used
+such as Jenkins, Argo Workflows, Flux, and Tekton.
+
+### task
+
+Keptn v1 defines some specific types of tasks,
+each of which is translated to KLT resources
+that are appropriate for the activity:
+
+* A **deployment task** becomes a
+  [Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/)
+  workload.
+  You can code
+  [KeptnTaskDefinition](../../yaml-crd-ref/taskdefinition.md)
+  and
+  [KeptnEvaluationDefinition](../../yaml-crd-ref/evaluationdefinition.md)
+  resources that are configured
+  to run either pre- or post-deployment tasks
+* An **evaluation task** becomes a
+  [KeptnEvaluationDefinition](../../yaml-crd-ref/evaluationdefinition.md)
+  resource.
+* All other standard tasks
+  (**action**, **approval**, **get-action**, **rollback**,
+  **release**, **test**)
+  as well as custom task types
+  that might be defined should be translated into
+  `KeptnTaskDefinition` resources.
+* The `key:value` **properties** for each Keptn v1 sequence
+  should be coded into the `KeptnTaskDefinition` resource
+  as appropriate.
+
+### SLIs
+
+Keptn v1
+[SLIs](https://keptn.sh/docs/1.0.x/reference/files/sli/)
+(Service Level Indicators)
+represent queries from the data provider
+such as Prometheus, Dynatrace, or Datadog,
+which is configured as a Keptn integration.
+
+When migrating to KLT, you need to define a
+[KeptnMetricsProvider](../../yaml-crd-ref/metricsprovider.md)
+resource for the data provider you are using.
+
+The queries defined for the Keptn v1 SLIs
+should be translated into
+[KeptnMetric](../../yaml-crd-ref/metric.md)
+resources.
+Note that KLT allows you to support multiple data providers
+and multiple instances of each data provider for your SLI's
+whereas Keptn v1 only allows you to use one SLI per project.
+
+### SLOs
+
+KLT at this time does not support the full range
+of Quality Gates evaluations that are represented by
+[SLOs](https://keptn.sh/docs/1.0.x/reference/files/slo/).
+Facilities such as weighting of SLI's and scoring of the evaluation
+do not currently exist.
+However, simple evaluations of an SLI can be defined as
+[KeptnEvaluationDefinition](../../yaml-crd-ref/evaluationdefinition.md)
+resources.
```

Comment:
If we merge this soonish, shall we point to a coming soon 😸 ? https://github.com/keptn/lifecycle-toolkit/issues/1646

Reply:
Done.